### PR TITLE
Fix AkkaProtocolStressTest spec

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs [hunk 1]
```diff
@@ -190,7 +192,12 @@ public void AkkaProtocolTransport_must_guarantee_at_most_once_delivery_and_messa
                         new FailureInjectorTransportAdapter.Drop(0.1, 0.1)));
             AwaitCondition(() => mc.IsCompleted && mc.Result, TimeSpan.FromSeconds(3));
 
-            var here = Here;
+            IActorRef here = null;
+            AwaitCondition(() =>
```
Comment (Arkatufus) [inline src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs:196]: `Here` property uses `ActorSelection` that can return a null `Subject` that causes NRE down the line (caused by `EmptyLocalActorRef`). Using `AwaitCondition` here to force the spec to wait until `Here` returns a valid `IActorRef`

Comment (Aaronontheweb) [reply src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs:196]: LGTM - how many times did you run this one successively without failures?

Comment (Arkatufus) [reply src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs:196]: 1000 times minimum

Comment (Aaronontheweb) [reply src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs:196]: That works for me

**Pull request merged**